src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/TransferClassificationActivityImpl.java

```diff
@@ -48,7 +48,7 @@ public TransferClassificationActivityImpl(ClassificationDao classificationDao,
 	public void classify(TransferSemanticKeyDTO transferSemanticKey) {
 		log.info("Transfer classification for organization id: {} and iuv: {}",
 			transferSemanticKey.getOrgId(), transferSemanticKey.getIuv());
-		if (!classificationDao.deleteTransferClassification(transferSemanticKey)) {
+		if (classificationService.deleteBySemanticKey(transferSemanticKey) == 0L) {
```

Comment:
this behavior is wrong! if the delete doesn't nothing, we could ignore it!
Can you remove the throw instruction?
At least, we could log as debug the number retrieved